### PR TITLE
updating Slovak translation files

### PR DESCRIPTION
**Pull request by jozefM923**

### CODE REVIEW

**Comment by eumagga0x2a** [comment]
The proposed translation update would restore the crash in encoding and in processing dialogs fixed by https://github.com/mean00/avidemux2_i18n/commit/b07a52f3011b4d4aae315eb5266a93d2297969f7 and thus cannot be merged.

In any case, please avoid regenerating translation file(s) on your own with options other than used in `qt_update_pro.sh:75`, especially we use `-locations relative`, else the diff becomes huge, obstructing review. If you need different options or want the translation file(s) to pick up the most recently added or modified translatable strings, please ping me on the Avidemux forum or open an issue here.

Regarding qtbase translation files, they should be ideally in sync with the upstream, but it looks like community contributions are quite complicated with Qt. I understand you taking an easy route. However, with focus shifting to Qt6, qtbase translations will need to be synced with upstream, discarding downstream changes.

I don't think updates for Qt4 translations (qt_*.ts) make much sense, hardly anyone still builds Avidemux with Qt4. Qt4 compatibility won't be maintained for long, I am inclined to drop it already once 2.8.1 is out (no official decision on this topic yet, however).

**Comment by jozefM923** [comment]
I'm sorry, but I use Poedit for translating and the change probably happened when converting the file back to Qt linguist.

File: avidemux_sk.ts [hunk 1]
```diff
@@ -3385,12 +3385,12 @@ Skúsiť to znovu?</translation>
     <message>
         <location filename="../../../avidemux_core/ADM_core/src/ADM_prettyPrint.cpp" line="+23"/>
         <source>%d minute(s)</source>
-        <translation>%d minút</translation>
+        <translation>%n minút</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:3388]: If merged, this translation, which reverts the fix I commited on Dec 20, 2021, will make Avidemux crash. Format specifiers like `%d`, `%u`, `%x`, `%s` must be preserved by translations, they are not interchangeable.

We cannot use numerus forms in source strings marked for translation by QT_TRANSLATE_NOOP macro, this is a limitation of the way how translation works in Avidemux. In a few cases where we do use numerus forms, we have to circumvent this infrastructure by using `QCoreApplication::translate()` directly, available only for Qt code (i.e. not in core).

File: avidemux_sk.ts [hunk 2]
```diff
@@ -3385,12 +3385,12 @@ Skúsiť to znovu?</translation>
...
     </message>
     <message>
         <location line="+7"/>
         <source>%d hour(s)</source>
-        <translation>%d hodín</translation>
+        <translation>%n hodín</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:3393]: See above.

File: avidemux_sk.ts [hunk 3]
```diff
@@ -6197,12 +6197,12 @@ Chcete ho načítať?
     <message>
         <location line="+1"/>
         <source>Adjust contrast, brightness, saturation and gamma.</source>
-        <translation>Upraví kontrast, jas, nasýtenie a gama.</translation>
+        <translation>Upraví kontrast, , sýtosť a gama.</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:6200]: Something deleted by accident?

File: avidemux_sk.ts [hunk 4]
```diff
@@ -561,7 +561,7 @@ Vypustiť časové údaje?</translation>
     <message>
         <location line="+5"/>
         <source>&amp;File</source>
-        <translation>&amp;Súbor</translation>
+        <translation>Súbor</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:564]: Please preserve the accelerator.

File: avidemux_sk.ts [hunk 5]
```diff
@@ -10726,47 +10728,47 @@ Only VP8/VP9/AV1 video and Vorbis/Opus audio supported</source>
...
         <location line="+56"/>
         <source>Average Bitrate:</source>
-        <translation type="unfinished"></translation>
+        <translation>Priemerná prenosová rýchlosť:</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>Pixel format:</source>
-        <translation type="unfinished"></translation>
+        <translation>_Formát pixelov:</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:10736]: Please fix the leading underscore. (Was it a bad suggestion by translation software? There is a different location in common code where we indeed use an accelerator as underscore, but here we are in Qt code where 1) an accelerator is `&amp;` and 2) we don't need any in this case.)

**Review by eumagga0x2a — COMMENTED** [review]
In any case, please close this PR and start afresh to make sure that huge bad changesets and their corrections don't remain in git logs.

Please also always create a branch rather than submitting patches from the master, thank you.

Last but not least, please clarify why we should add Slovak translation for Qt4 qtbase despite building Avidemux with Qt4 being deprecated and Qt4 support to be dropped soon.

**Pull request closed without merging**